### PR TITLE
werckerによるRubocopとPRへの指摘を行う

### PR DESCRIPTION
**Pull request by seoky0**

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+  def index
+    end
```
Comment (seoky0) [inline app/controllers/home_controller.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Layout/DefEndAlignment: end at 3, 4 is not aligned with def at 2, 2.

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
+class HomeController < ApplicationController
+  def index
+  end
+
+  def show
```
Comment (seoky0) [inline app/controllers/home_controller.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/AbcSize: Assignment Branch Condition size for show is too high. [64/15]

Comment (seoky0) [inline app/controllers/home_controller.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/MethodLength: Method has too many lines. [64/10]

File: app/controllers/home_controller.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  def index
+    puts 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
```
Comment (seoky0) [inline app/controllers/home_controller.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/LineLength: Line is too long. [182/128]

File: app/controllers/home_controller.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+class HomeController < ApplicationController
+  def index; end
+
+  def new
```
Comment (seoky0) [inline app/controllers/home_controller.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/MethodLength: Method has too many lines. [13/10]

**Comment by kyohei0423** [comment]
コメント

**Comment by kyohei0423** [comment]
コメント2

**Comment by kyohei0423** [comment]
hoge

**Comment by kyohei0423** [comment]
コミット

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,17 @@
 module ApplicationHelper
+  def hoge_helper
```
Comment (seoky0) [inline app/helpers/application_helper.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/MethodLength: Method has too many lines. [13/10]

File: app/controllers/home_controller.rb [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    puts 'hoge'
+    puts 'hoge'
+    puts 'hoge'
+    puts 'hoge'
+    puts 'hoge'
+    puts 'hoge'
+    puts 'hoge'
+  end
+
+  def hoge
```
Comment (review-master) [inline app/controllers/home_controller.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/MethodLength: Method has too many lines. [13/10]

**Pull request merged**